pkg/util/caller/resolver.go

```diff
@@ -120,7 +119,7 @@ func (cr *CallResolver) Lookup(depth int) (file string, line int, fun string) {
 		if len(matches) == 1 {
 			file = matches[0]
 		} else {
-			file = filepath.Join(matches[1:]...)
+			file = path.Join(matches[1:]...)
```

Comment:
I'm not sure it's appropriate to add that comment; this is patently the difference between the two packages.

If I added this comment, where exactly would you like to see it?